inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/RecommendationEditorExtension.java

```diff
@@ -242,7 +242,12 @@ private void actionRejectRecommendation(AnnotationActionHandler aActionHandler,
     }
     
     @Override
+<<<<<<< HEAD
```

Comment:
Unresolved merge conflict here. The first parameter needs to be `CAS aCas`.

Reply:
Oh no, sorry, I currently cannot compile the project, so I didn't notice it...